root/static/scripts/release-editor/fields.js

```diff
@@ -977,7 +977,7 @@ class Release extends mbEntity.Release {
     this.packagingID = ko.observable(data.packagingID);
     this.barcode = new Barcode(data.barcode);
     this.comment = ko.observable(data.comment);
-    const annotationText = data.latest_annotation
+    const annotationText = data.latest_annotation?.text
       ? data.latest_annotation.text
       : '';
```

Comment:
```suggestion
    const annotationText = data.latest_annotation?.text ?? '';
```
Could be shortened a bit since this will always be a string, null or undefined. Thanks for finding and fixing this, by the way. (I should really try to set up my own local development server soon, just need to find a partition with enough free space and a bit of time...)